website/content/docs/server/install.mdx

```diff
@@ -38,9 +46,9 @@ chart can be used to deploy Waypoint server directly onto a Kubernetes cluster w
 We recommend reading the README detailed on that project for more information on
 how to get going using Helm and Waypoint.
 
-You can also install Waypoint server onto Kubernetes using the `waypoint install`
-command. This command sets up a Service, Statefulset, Deployment, and Replicaset
-and deploys Waypoint server and its runner as two separate pods by default.
+Installing the Waypoint server to Kubernetes using the `waypoint install` command
+will also use the Waypoint Helm chart for the installation, setting sensible default
+values for the chart values.
```

Comment:
Above here we say "Installing Waypoint server onto Kubernetes has a couple of options: Using the
waypoint-helm chart or using `waypoint install`.", I think we should probably remove that part now and combine what's left with this new sentence, what do you think?

Reply:
@catsby do you mean removing the presentation of the option to directly install with Helm via `helm install`?

Reply:
I think we should reword or restructure this part to mention/include that `waypoint install` uses the helm chart, but they could also use it directly if they wanted. Right now it's worded as either or, but `waypoint install` uses the helm chart